docs/walkthrough/walkthrough-istio.md

```diff
@@ -0,0 +1,384 @@
+EOF
+```
+
+After a short delay, the service mesh pods should be up and running in the `istio-system` namespace, running the command `kubectl get pod -n istio-system` should display something similar to the following output:
+
+```bash
+NAME                                   READY   STATUS    RESTARTS   AGE
+grafana-5dc4b4676c-vh9r6               1/1     Running   0          114s
+istio-egressgateway-65d5579779-6sbpr   1/1     Running   0          2m
+istio-ingressgateway-96f44fb4d-wwsq8   1/1     Running   0          2m
+istio-tracing-8584b4d7f9-l7hnp         1/1     Running   0          114s
+istiod-7988b4d788-2vq9k                1/1     Running   0          2m15s
+kiali-6f457f5964-np7hf                 1/1     Running   0          114s
+prometheus-67988db74f-2x2hb            2/2     Running   0          114s
+```
+
+## Note on sidecar injection
+
+Istio supports two modes for injecting the service mesh proxy sidecar:
+- manual injection with `istioctl kube-inject`
+- automatic injection using a mutating webhook
+
+In this walkthrough, the manual injection technique will be used.
+
+You can turn on automatic sidecar injection by adding the `istio-injection=enabled` label on the `tekton-pipelines` namespace and restarting all pods once all Tekton components have been deployed:
+
+```bash
+kubectl label namespace tekton-pipelines istio-injection=enabled
+
+kubectl delete pods --all -n tekton-pipelines
+```
+
+## Installing Tekton Pipelines
+
+Tekton Dashboard requires to have Tekton Pipelines installed.
+
+Installing the latest Tekton Pipelines release is done by running the following command:
+
+```bash
+curl -sL https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml | istioctl kube-inject -f - | kubectl apply -f -
+
+kubectl wait -n tekton-pipelines \
+  --for=condition=ready pod \
+  --selector=app.kubernetes.io/part-of=tekton-pipelines,app.kubernetes.io/component=controller \
+  --timeout=90s
+```
+
+Tekton Pipelines should be up and running.
+
+## Installing Tekton Triggers (optional)
+
+Installing Tekton Triggers is not mandatory, if you don't want to install Tekton Triggers, you can skip this section.
+If you don't install Tekton Triggers, triggers related resources won't show up in the side nav.
+
+Installing the latest Tekton Triggers release is done by running the following command:
+
+```bash
+curl -sL https://storage.googleapis.com/tekton-releases/triggers/latest/release.yaml | istioctl kube-inject -f - | kubectl apply -f -
+
+kubectl wait -n tekton-pipelines \
+  --for=condition=ready pod \
+  --selector=app.kubernetes.io/part-of=tekton-triggers,app.kubernetes.io/component=controller \
+  --timeout=90s
+```
+
+Tekton Triggers should be up and running.
+
+## Installing Tekton Dashboard
+
+Installing the latest Tekton Dashboard release is done by running the following command:
+
+```bash
+curl -sL https://raw.githubusercontent.com/tektoncd/dashboard/master/scripts/release-installer | \
```

Comment:
fails here:

```shell
$ curl -sL https://raw.githubusercontent.com/tektoncd/dashboard/master/scripts/release-installer | \
→    bash -s -- build latest --output /dev/stdout | istioctl kube-inject -f - | kubectl apply -f -
Error: error converting YAML to JSON: yaml: line 2: mapping values are not allowed in this context
error: no objects passed to apply
```

Reply:
yeah, we don't have cut a release that support version `latest` yet (and `v0.8.2` won't work here because you can't | it to `istioctl`).

Reply:
ah of course, that makes sense now 👍